File: CPP/loops5.cpp
// sum of first N natural numbers squares
/*#include<iostream>
using namespace std;
int main()
{
    int n, sum  = 0;
    cout<<"ENTER N : ";
    cin>>n;
    for(int i = 1; i<=n; i++){
        sum += i*i;

        cout<<"sum of first N natural numbers squares  : "<<sum<<endl;

    }
    return 0;
    
} */

#include <iostream>
using namespace std;

int main() {
    int n, sum = 0;
    cout << "Enter n: ";
    cin >> n;

    for (int i = 1; i <= n; i++) {
        sum += i * i;
        if (i < n)
            cout << i << "^2 + ";
        else
            cout << i << "^2 = ";
    }

    cout << sum << endl;
    return 0;
}

File: Java/loops5.java
// Sum of squares of first N natural numbers (series + result)
import java.util.Scanner;

public class loops5 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter N: ");
        int n = sc.nextInt();
        int sum = 0;

        for (int i = 1; i <= n; i++) {
            sum += i * i;
            if (i < n) {
                System.out.print(i + "^2 + ");
            } else {
                System.out.print(i + "^2 = ");
            }
        }
        System.out.println(sum);
    }
}
